libs/core/src/components/pds-input/pds-input.scss

```diff
@@ -50,10 +50,10 @@
   --field-padding-block: 8px;
   --field-padding-inline: 16px;
 
-  --hint-font-size: 12px;
-  --hint-font-weight: 400;
-  --hint-line-height: 20px;
-  --hint-margin-top: 8px;
+  --helper-font-size: 12px;
+  --helper-font-weight: 400;
+  --helper-line-height: 20px;
+  --helper-margin-top: 8px;
```

Comment:
why wouldn't this be `helper-message` it would follow the pattern shown below?

Reply:
Done